### ISSUE TITLE
Kubernetes Helm Chart

### ISSUE THREAD

**Issue by MatthiasLohr** [roundcube/roundcubemail-docker#161]
Hey folks,

thanks for providing this docker image! It helped me a lot to get Roundcube running on my Kubernetes cluster. Maybe it's out of scope, but without asking we never find out:

I wrote a helm chart to deploy roundcube to a Kubernetes cluster. Maybe it is worth mentioning this in the documentation? Link to the helm chart is here:
  * https://gitlab.com/MatthiasLohr/roundcube-helm-chart -- source code
  * https://artifacthub.io/packages/helm/mlohr/roundcube -- chart website on ArtifactHub

Best regards
Matthias

**Comment by thomascube** [comment]
Hello Matthias! Thanks for creating and sharing this with us. There's already another Helm chart for Roundcube that we're aware of: https://github.com/volker-raschek/roundcube-charts and after I found some time to have a closer look, I'll add the links to the roundcube.net website. Maybe you also found our [Kubernetes example](https://github.com/roundcube/roundcubemail-docker/blob/master/examples/kubernetes.yaml) and you might have some suggestions or additions to improve it.

**Comment by MatthiasLohr** [comment]
I left some feedback (sorry for my honesty) in #162. As written there: If you want, I can provide a couple of lines of code as an example for how to get Roundcube running using a helm chart. So people have a fast and easy start with Roundcube on Kubernetes, but they can then just continue using the chart and ajusting more and more configuration values over time. What do think about that?

For the helm chart available at https://github.com/volker-raschek/roundcube-charts: I was in contact with the author and asked for collaboration, because I think the world needs one (ideally) perfect chart more than two half perfect charts. However, after the first reply, I didn't get any further replies (yet). So I'm trying to improve my helm chart as far as time allows for it. There are also a couple of features, which I have implemented, which are not (yet) implemented in the other helm chart. However, things still open in my chart:
  * Allow for direct deployment of a database using existing and well-validated and well-maintained helm charts (e.g. Bitnami).
  * Adding a side car for automatic plugin installation before roundcube is started (plugin configuration is already completely available the helm way of life, but currently the actual plugin has to be provided using an self-built image).

**Comment by thomascube** [comment]
Hello again. Your Helm chart looks really good and I just installed it with minikube and a mysql chart. It's also mentioned in a news post on roundcube.net and in the download section of our website.

> things still open in my chart:
> * Allow for direct deployment of a database using existing and well-validated and well-maintained helm charts (e.g. Bitnami).

Some kind of hint or a few lines of code how to set up a database would really be helpful.

> * Adding a side car for automatic plugin installation before roundcube is started

Yes, I played around with `initContainers` a bit but so far I wasn't successful. But that should be the way to install additional plugins. How to do this is explained in the [examples README](/roundcube/roundcubemail-docker/tree/master/examples#installing-roundcube-plugins).

We're definitely looking for updates on your charts and we're happy for your contributions.
